python/GafferSceneUI/SetUI.py

```diff
@@ -42,6 +42,31 @@
 import GafferScene
 import GafferSceneUI
 
+## Menu Presentation
+# -----------------
+# Many facilities have large numbers of sets with structured names. When
+# presented in a flat list, it can be hard to navigate. The path function
+# allows the name of a set to be amended or turned into a hierarchical menu
+# path (ie, by returning a string containing '/'s) - creating sub-menus
+# wherever Gaffer displays a list of sets.
+
+__menuPathFunction = lambda n : n
+
+## f should be a callable that takes a set name, and returns a menu path
+# @see IECore.Menu
```

Comment:
IECore.MenuDefinition

Reply:
Ahh yes, sorry, always forget `Menu` is in Gaffer. Fixed [here](https://github.com/GafferHQ/gaffer/pull/4098/files#diff-247255cb8d3cc8071623d404dc1e2e4fad247bc56fbf36b9c900ea69b9989df7R55-R56).